### PR TITLE
[Review] Request from '' @ 'yast/ycp-killer/readme-improvements'

### PR DESCRIPTION
**Pull request by dmajda**
Please review the following changes:
- 1ebc262 README.md: Add the "Overview" section
- 189fd98 README.md: Move & rename the "Resulting Directory Structure" section
- 78c0e0f README.md: Improve step 2 in the installation a bit
- 3a2a630 README.md: Simplify the introduction

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,34 +125,69 @@ things get gemified, packaged, etc.
...
-    ├── control-rnc    ->  /usr/share/YaST2/schema/control/rnc
-    ├── desktop        ->  /usr/share/applications/YaST2
-    └── fillup         ->  /var/adm/fillup-templates
-```
+  * Clone YaST module Git repository.
+  * Restructure the YaST module source code to match the new structure (see [New
+    YaST Module Structure](#new-yast-module-structure).
+  * Apply patches to the restructured YaST module source code (typically to
+    adapt Makefiles to Ruby translation and to work around Y2R deficiencies).
+  * Compile YaST module's YCP modules (without this any code depending on them
```
Comment (lslezak) [inline README.md:143]: `module's YCP modules` looks strange to me, `module's YCP files` would be better IMHO.

Comment (lslezak) [inline README.md:143]: Uh, it's actually true, we compilr YCP modules only, I was just confused by overloaded `module` term...

File: README.md [hunk 2]
```diff
@@ -134,34 +125,69 @@ things get gemified, packaged, etc.
...
+    can't be translated by Y2R).
+  * Convert the YaST module source code into Ruby.
+  * Create a YaST module package source.
```
Comment (lslezak) [inline README.md:148]: I miss the Makefile generation step.

File: README.md (hunk 1 above)
Comment (dmajda) [inline README.md:143]: Yeah, it's confusing. I try at least to qualify "module" with either "YaST" or "YCP" in any place where confusion can arise.

File: README.md (hunk 2 above)
Comment (dmajda) [inline README.md:148]: Fixed in bf1951ac1e8270168de1052080ff0e1fa4331b06.

**Pull request merged**